cpp/perspective/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(psp)
 include(CheckCCompilerFlag)
 
 set(CMAKE_BUILD_TYPE "Release")
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
should update/remove gnu flag to match: https://github.com/finos/perspective/blob/88e08788340652e614c4755b7d6830ee64869ddf/cpp/perspective/CMakeLists.txt#L414

Reply:
Where is this tested?